src/main/java/me/kavin/piped/server/ServerLauncher.java

```diff
@@ -403,6 +402,18 @@ AsyncServlet mainServlet(Executor executor) {
                     } catch (Exception e) {
                         return getErrorResponse(e, request.getPath());
                     }
+                })).map(POST, "/user/playlists/description/change", AsyncServlet.ofBlocking(executor, request -> {

```

Comment:
What if we make this a `PATCH` method, with the URI path just being `/user/playlists/description`?